platform/core/src/classes/MetadataProvider.ts

```diff
@@ -141,15 +141,16 @@ class MetadataProvider {
           seriesInstanceUID: instance.SeriesInstanceUID,
           seriesNumber: toNumber(instance.SeriesNumber),
           studyInstanceUID: instance.StudyInstanceUID,
+          seriesDescription: instance.SeriesDescription,
           seriesDate,
           seriesTime,
         };
         break;
       case WADO_IMAGE_LOADER_TAGS.PATIENT_STUDY_MODULE:
         metadata = {
-          patientAge: toNumber(instance.PatientAge),
-          patientSize: toNumber(instance.PatientSize),
-          patientWeight: toNumber(instance.PatientWeight),
+          patientAge: instance.PatientAge,
```

Comment:
not sure why we are removing the toNumber, i know lots of issues arise from not having number or not having string etc etc

Reply:
Mostly it gives NaN as a response.  If we want a formatted version, then we should create secondary computations that operate on the base/default data.  There are a whole bunch of cases we return garbage data from the metadata provider, and if we find particular formatting issues, then I will create
patientFormatted
as a module name that calls patientModule and does the formatting.